backend/src/db.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @desc This file connects the api to the database and exports a
+ * simple query module for accessing it's data
+ * @author @GDamaso
+ */
+import pg from 'pg';
+
+const { Client } = pg;
+
+const client = new Client({
+  user: process.env.POSTGRES_USER,
+  password: process.env.POSTGRES_PASSWORD,
```

Comment:
If you ran this in a local terminal instead of docker it would fail because the `.envs` don't match what docker is using, this would register as 'undefined'.